examples/non-fungible-token/tests/workspaces/test_approval.rs

```diff
@@ -87,20 +85,17 @@ async fn simulate_simple_approve(
     assert!(alice_approval_id_is_2);
 
     // approving another account gives different approval_id
-    let res = nft_contract
+    nft_contract
         .call("nft_approve")
-        .args_json((
-            TOKEN_ID,
-            token_receiver_contract.id(),
-            Option::<String>::None,
-        ))
+        .args_json((TOKEN_ID, token_receiver_contract.id(), Option::<String>::None))
         .max_gas()
         // note that token_receiver's account name is shorter, and so takes less bytes to store and
         // therefore requires a smaller deposit!
-        .deposit(NearToken::from_yoctonear(450000000000000000000))
+        .deposit(NearToken::from_yoctonear(550000000000000000000))
         .transact()
-        .await?;
-    assert!(res.is_success());
```

Comment:
nit: previous assert! was more readable and prominent it's a test assetion than an unwrap